client/widmgr/mock.go

```diff
@@ -129,7 +133,7 @@ func (m MockWIDMgr) Run() error { return nil }
 func (m MockWIDMgr) Get(identity structs.WIHandle) (*structs.SignedWorkloadIdentity, error) {
 	sid, ok := m.swids[identity]
 	if !ok {
-		return nil, fmt.Errorf("identity not found")
+		return nil, fmt.Errorf("identity name %s for workload %s not found", identity.IdentityName, identity.WorkloadIdentifier)
```

Comment:
Just in case we want to write a test that checks for specific errors, maybe this message should exactly match the one returned from the real implementation? https://github.com/hashicorp/nomad/blob/main/client/widmgr/widmgr.go#L143

Reply:
good thinking